### ISSUE TITLE
models/research/slim/nets/inception_v2.py: Missing batch normalizations after conv1

### ISSUE THREAD

**Issue by mefathy** [tensorflow/models#4843]
Attempts to load the provided model checkpoint for inception_v2 fails complaining that the checkpoint does not contain values for the biases of the various convolutions defined in the network. Inspecting the checkpoint file and the graph generated by inception_v2.py reveals that the checkpoint file contains batch-normalization for convolutions starting from conv2 and no bias except at the conv1 layer (which does not have a following batch normalization). The graph generated by inception_v2.py does not show any batch normalization layers.

Since the inception_v2 is supposed to have batch normalization, I think the issue here is that inception_v2.py should provide batch normalization parameters to the slim.conv2d invocations after conv1 so that slim adds batch normalization and avoids defining biases.

**Comment by tensorflowbutler** [comment]
Thank you for your post. We noticed you have not filled out the following field in the issue template. Could you update them if they are relevant in your case, or leave them as N/A? Thanks.
What is the top-level directory of the model you are using
Have I written custom code
OS Platform and Distribution
TensorFlow installed from
TensorFlow version
Bazel version
CUDA/cuDNN version
GPU model and memory
Exact command to reproduce

**Comment by tensorflowbutler** [comment]
Hi There,
 We are checking to see if you still need help on this, as this seems to be considerably old issue. Please update this issue with the latest information, code snippet to reproduce your issue and error you are seeing.
 If we don't hear from you in the next 7 days, this issue will be closed automatically. If you don't need help on this issue any more, please consider closing this.
